src/Symfony/Component/DependencyInjection/Tests/Fixtures/includes/autowiring_classes.php

```diff
@@ -198,7 +198,7 @@ public function __construct(A $a, $foo = 'default_val', Lille $lille)
     }
 }
 
-/*
+/**
  * Classes used for testing createResourceForClass
  */
 class ClassForResource
```

Comment:
These docs were incorrectly interpretted by PHP CS Fixer as single line, thus fixing them to Doc Blocks